main.py

```diff
@@ -48,6 +48,7 @@ def add_commands(up: Updater) -> None:
         BotCommand("urp", "URP"),
         BotCommand("mensa", "orari e menù della mensa"),
         BotCommand("biblioteca", "orari della biblioteca"),
+        BotCommand("aule_studio", "orari delle aule studio", "dove si trova"),
```

Comment:
penso che devi dargli solo due valori in input, quindi `("aule_studio", "tutta la descrizione qui")`

Reply:
ok perfetto

Reply:
risolvo